src/main/java/com/github/cchacin/cucumber/steps/DatabaseSteps.java

```diff
@@ -95,4 +105,37 @@ public void I_have_the_following_sql_script(final String script) throws Throwabl
         new ScriptRunner(this.destination.getConnection())
                 .runScript(new BufferedReader(new FileReader(Thread.currentThread().getContextClassLoader().getResource(script).getPath())));
     }
+
+    @Then("^I should have the following rows in the \"([^\"]*)\" table:$")
+    public void I_have_the_following_rows_in_the_table(
```

Comment:
fixed in https://github.com/xsuoownit/cucumber-common-steps/commit/2ee35d2b82cfaa5f0a4395abd508e07301ec318e